src/api/routes/songs.ts

```diff
@@ -3,10 +3,53 @@ const data = require("../../../data/songdata.json");
 
 const router = express.Router();
 
+const fetchData=(key:string,value:string)=>{
```

Comment:
Don't create another method. We already have a [`filter()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method for such things.

Reply:
@ShauryaAg Done!